### ISSUE TITLE
Cannot add PPA: 'ppa:~cowsql/ubuntu/master'.

### ISSUE THREAD

**Issue by muslu** [cowsql/cowsql#17]
```
root@licus:/tmp/incus# add-apt-repository ppa:cowsql/master
Cannot add PPA: 'ppa:~cowsql/ubuntu/master'.
The team named '~cowsql' has no PPA named 'ubuntu/master'
Please choose from the following available PPAs:
 * 'main':  main cowsql builds
 * 'stable':  stable cowsql builds

```

**Comment by muslu** [comment]
```
root@licus:/tmp/incus# add-apt-repository ppa:cowsql/stable
 Package builds of the latest stable release of cowsql.
 More info: https://launchpad.net/~cowsql/+archive/ubuntu/stable
Press [ENTER] to continue or Ctrl-c to cancel adding it.

Hit:1 http://tr.archive.ubuntu.com/ubuntu focal InRelease
Get:2 http://tr.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Ign:3 http://ppa.launchpad.net/cowsql/stable/ubuntu focal InRelease
Hit:4 http://tr.archive.ubuntu.com/ubuntu focal-backports InRelease
Err:5 http://ppa.launchpad.net/cowsql/stable/ubuntu focal Release
  404  Not Found [IP: 185.125.190.80 80]
Hit:6 http://tr.archive.ubuntu.com/ubuntu focal-security InRelease
Get:7 http://tr.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1,159 kB]
Get:8 http://tr.archive.ubuntu.com/ubuntu focal-updates/universe Translation-en [278 kB]
Reading package lists... Done                                   
E: The repository 'http://ppa.launchpad.net/cowsql/stable/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

```

**Comment by freeekanayaka** [comment]
> ```
> root@licus:/tmp/incus# add-apt-repository ppa:cowsql/master
> Cannot add PPA: 'ppa:~cowsql/ubuntu/master'.
> The team named '~cowsql' has no PPA named 'ubuntu/master'
> Please choose from the following available PPAs:
>  * 'main':  main cowsql builds
>  * 'stable':  stable cowsql builds
> ```

As the help message says, the available PPAs are `main` (tracking gith HEAD) and `stable` (tracking the latest release).

Btw, where did you find this PPA? It's not advertised in the README, since it's used mainly for development. It's find if you want to use it though.

**Comment by freeekanayaka** [comment]
> Btw, where did you find this PPA? It's not advertised in the README, since it's used mainly for development. It's find if you want to use it though.

Amend: the PPA is actually referenced by the README, and was fixed by #18.

**Comment by muslu** [comment]
thank you and you team.

respect.
